app/models/character.rb

```diff
@@ -54,7 +54,7 @@ def valid_template
       errors.add(:template, "must be yours") if template.present? && template.user_id != user.id
       return
     end
-    @template = Template.new(user: user, name: new_template_name)
+    @template = Template.new(name: new_template_name, user: user)
```

Comment:
this change is due to an intermediate thing I meant to revert (but ended up reverting in reverse I guess?), oops. shouldn't affect anything